src/components/Searchbar/Searchbar.js

```diff
@@ -25,12 +24,16 @@ const MyInput = styled.input`
 	font-size: 1.1em;
 `;
 
-export default function Searchbar() {
+export default function Searchbar({searchTerm, onSearchTermChange}) {
 	return (
 		<Wrapper>
 			<MySearchbar>
 				<SearchRoundedIcon color="action" />
-				<MyInput placeholder="Enter your location here" />
+				<MyInput
```

Comment:
Your `input` element has no `label`, but should have one or at least an `aria-label`. Your `MySearchbar` component should be a `form` element and not a `div` as your search is a form. I also have two naming remarks:
The name of your component is already `Searchbar`. Therefore, I would choose a different name for `MySearchbar`. As this should be the `form` element, `Form` would be my choice.
The word `My` is also nothing that helps to understand your code. `Input` as name would be my choice.
Please think about this. :)